Magento2/Sniffs/Commenting/ClassPropertyPHPDocFormattingSniff.php

```diff
@@ -125,23 +126,28 @@ public function processMemberVar(File $phpcsFile, $stackPtr)
             null,
             false
         );
-        if ($this->PHPDocFormattingValidator->providesMeaning(
-            $isShortDescriptionPreviousVar,
-            $commentStart,
-            $tokens
-        ) !== true) {
-            preg_match(
-                '`^((?:\|?(?:array\([^\)]*\)|[\\\\\[\]]+))*)( .*)?`i',
-                $tokens[($foundVar + 2)]['content'],
-                $varParts
-            );
-            if ($varParts[1]) {
-                return;
-            }
+
+        if (stripos($tokens[$isShortDescriptionPreviousVar]['content'], $tokens[$string]['content']) !== false) {
```

Comment:
Please check that $isShortDescriptionPreviousVar is not false before using it as an array key